server.js

```diff
@@ -138,6 +139,33 @@ app.get("/*", (req, res) => {
   res.sendFile(path.join(__dirname, "dist", "storyline", "index.html"));
 });
 
+//CAPTCHA
+app.post('/api/captcha', async (req,res) => {
```

Comment:
Not sure if i used sync properly. kinda confused on that stuff, just saw everything else in server use it so i put it in